pkgs/development/libraries/graphene/default.nix

```diff
@@ -26,12 +28,21 @@ stdenv.mkDerivation rec {
     owner = "ebassi";
     repo = pname;
     rev = version;
-    sha256 = "v6YH3fRMTzhp7wmU8in9ukcavzHmOAW54EK9ZwQyFxc=";
+    sha256 = "P6JQhSktzvyMHatP/iojNGXPmcsxsFxdYerXzS23ojI=";
   };
 
   patches = [
     # Add option for changing installation path of installed tests.
     ./0001-meson-add-options-for-tests-installation-dirs.patch
+
+    # Disable flaky simd_operators_reciprocal test
```

Comment:
Is it really disabling the test?

Reply:
Actually I think it disables the test (https://github.com/ebassi/graphene/commit/4fbdd07ea3bcd0964cca3966010bf71cb6fa8209#diff-b01d1a458f700e8df74d5d349a45de62f1f36082aee97dc2a90a0a785b925edf), and ofborg says this fixes the darwin build.

Reply:
Oh, I did not notice the revert.